dev/test/specs/starter-test.js

```diff
@@ -380,4 +382,40 @@ describe('test Arcs demo flows', function() {
 
     browser.close();
   });
+
+  it('can use an arc with the default global manifests', function() {
+    initTestWithNewArc();
+
+    // remove solo from our URL to use the default
+    const url = new URL(browser.getUrl());
+    url.searchParams.delete('solo');
+    browser.url(url.href);
+
+    // load our utils in the new page
+    loadSeleniumUtils();
+
+    waitForStillness();
+    browser.waitUntil(
+      () => {
+        const allSuggestions = pierceShadows([
```

Comment:
const allSuggestions = pierceShadows([
      'div[slotid="suggestions"]',
      'suggestion-element'
]);
if (!allSuggestions.value || 0 == allSuggestions.value) {
       return false;
}

These line have a duplicate in acceptSuggestion method.. maybe create a helper method?